lib/orderbook/errors.ts

```diff
@@ -14,6 +14,7 @@ const errorCodes = {
   QUANTITY_DOES_NOT_MATCH: codesPrefix.concat('.10'),
   CURRENCY_MISSING_ETHEREUM_CONTRACT_ADDRESS: codesPrefix.concat('.11'),
   INSUFFICIENT_OUTBOUND_BALANCE: codesPrefix.concat('.12'),
+  MIN_QUANTITY_VIOLATED: codesPrefix.concat('.13'),
```

Comment:
We should add this error to the list of errors that result in a `INVALID_ARGUMENT` grpc error in GrpcService.ts.